src/components/Filter/Toggle/index.tsx

```diff
@@ -0,0 +1,66 @@
+import { useState } from "react";
+import styled from "@emotion/styled";
+import Button, { Variant } from "@leafygreen-ui/button";
+import { size } from "constants/tokens";
+
+export enum ToggleButton {
+  Left = "left",
+  Right = "right",
+}
+
+interface ToggleProps {
+  activeButton?: ToggleButton.Left | ToggleButton.Right;
+  ["data-cy"]?: string;
+  leftText: string;
+  rightText: string;
+}
+
+const Toggle: React.FC<ToggleProps> = ({
+  activeButton = "left",
+  "data-cy": dataCy,
+  leftText,
+  rightText,
+}) => {
```

Comment:
I think making this a controlled component would be the easiest to use.

So adding a `value` and `onChange` prop. And using those to manage its state.